external-report-demo/template.yml

```
@@ -0,0 +1,41 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  report-service-demo-report
+  Sample SAM Template for report-service-demo-report
+
+Parameters:
+  JwtHmacSecret:
+    Type: String
+    Description: Secret to verify request
+    # Default: [for development, the JwtHmacSecret can be pasted here]
```

```suggestion
    # Default: [for development, the JwtHmacSecret can be pasted here, BUT SHOULD NEVER BE COMMITTED]
```
May be obvious, but in such situations stating the obvious may not be a bad thing. 😉